app/controllers/photoTagController.js

```diff
@@ -1,7 +1,35 @@
+import { PhotoTagCreate, PhotoTagResponse } from "../models/apiModels/photoTag.js";
+import PhotoTagAccessor from "../databaseAccessors/photoTagAccessor.js";
+import { ErrorRepeatedTagName, ErrorUnexpected, HttpError } from "../error/errors.js";
+
+
 /**
  * PhotoTagController Class
  *
  * This class controls the behaviour of any web request
  * related to PhotoTags
  */
-export default class PhotoTagController {}
+export default class PhotoTagController {
+  /**
+   * Creates a PhotoTag with ref fields populated
+   *
+   * @param {Request} req
+   * @param {Response} res
+   */
+  static async create(req, res) {
+    try {
+      const createdTag = new PhotoTagCreate(req.body);
+      const newTag = await PhotoTagAccessor.createPhotoTag(createdTag);
+      const updateNewTag = await PhotoTagAccessor.getTagByID(newTag._id);
+      const newTagResponse = new PhotoTagResponse(updateNewTag.toObject());
+      
+      res.status(201).json(newTagResponse); 
```

Comment:
I am concerned about the naming here. `updateNewTag` seems like a misleading name for the populated tag, especially if it's just coming from a get function. Furthermore, `createdTag` is also misleading because the tag isn't actually created by the time it reaches that line. I would rename to something similar to as follows: 
```suggestion
      const tagCreate = new PhotoTagCreate(req.body);
      const newTag = await PhotoTagAccessor.createPhotoTag(tagCreate);
      const populatedNewTag = await PhotoTagAccessor.getTagByID(newTag._id);
      const newTagResponse = new PhotoTagResponse(populatedNewTag.toObject());
      
      res.status(201).json(newTagResponse); 
```